pkg/datarepair/checker/checker.go

```diff
@@ -63,13 +74,34 @@ func NewChecker(metainfo *metainfo.Service, repairQueue queue.RepairQueue, overl
 func (checker *Checker) Run(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	return checker.Loop.Run(ctx, func(ctx context.Context) error {
-		err := checker.IdentifyInjuredSegments(ctx)
+	c := make(chan error)
+
+	go func() {
+		c <- checker.Loop.Run(ctx, func(ctx context.Context) error {
+			err := checker.IdentifyInjuredSegments(ctx)
+			if err != nil {
+				checker.logger.Error("error with injured segments identification: ", zap.Error(err))
+			}
+			return nil
+		})
+	}()
+
+	go func() {
+		c <- checker.IrrepairLoop.Run(ctx, func(ctx context.Context) error {
+			err := checker.irrepairProcess(ctx)
+			if err != nil {
+				checker.logger.Error("process", zap.Error(err))
```

Comment:
just `process` message doesn't mean anything. Set meaningful message for error.

Reply:
done